src/Screen/Fields/Cropper.php

```diff
@@ -65,7 +67,8 @@ class Cropper extends Picture
      * @var array
      */
     protected $inlineAttributes = [
-        'accept',
+        '
+        @method Picture acceptedFiles(string $value = null)
```

Comment:
Check it out, I think there is a typo here

Reply:
I've fixed this